packages/test/test-end-to-end-tests/src/test/fluidObjectHandle.spec.ts

```diff
@@ -29,14 +29,14 @@ describeFullCompat("FluidObjectHandle", (getTestObjectProvider) => {
 
     beforeEach(async () => {
         // Create a Container for the first client.
-        const firstContainer = await provider.makeTestContainer();
+        const firstContainer = await provider.makeTestContainer({ useLegacyContainerRuntime: true });
         firstContainerObject1 = await requestFluidObject<ITestDataObject>(firstContainer, "default");
         const containerRuntime1 = firstContainerObject1._context.containerRuntime;
         const dataStore = await containerRuntime1.createDataStore(TestDataObjectType);
         firstContainerObject2 = await requestFluidObject<ITestDataObject>(dataStore, "");
 
         // Load the Container that was created by the first client.
-        const secondContainer = await provider.loadTestContainer();
+        const secondContainer = await provider.loadTestContainer({ useLegacyContainerRuntime: true });
```

Comment:
I'm sure we will have more versions of "legacy" here over time. I'd rather name this property based on what you get / looking for - a container runtime with global agent scheduler.
Ideally these things (in future) would be composable, such that I could get this behavior with some other optional behavior

Reply:
Sounds good, will change names.

These are intended to have a short lifespan (I intend to remove the legacy helpers in N+2), since customers can easily instantiate `AgentScheduler`s on their own in their container code, so I'm not making it a goal to encourage composition with the legacy helpers.